### ISSUE TITLE
Typo in *Web Accessiblity Checklist* title

### ISSUE THREAD

**Issue by chrisnager** [a11yproject/a11yproject.com#32]
Accessibility is missing an _i_ in the title _Web Accessibility Checklist_.
